Extensions/find_inactives.js

```diff
@@ -1,8 +1,8 @@
 //* TITLE Find Inactives **//
-//* VERSION 0.2.1 **//
+//* VERSION 0.3.0 **//
 //* DESCRIPTION Find the inactive blogs you follow **//
-//* DEVELOPER STUDIOXENIX **//
-//* DETAILS This extension lets you find the blog that haven't been updated for over 30 days. Just go to list of blogs you follow, then click on &quot;Find Inactive Blogs&quot; button below your Crushes to get started. **//
+//* DEVELOPER new-xkit **//
+//* DETAILS This extension lets you find the blog that haven't been updated for a certain amount of time. Just go to list of blogs you follow, then click on &quot;Find Inactive Blogs&quot; button below your Crushes to get started. **//
```

Comment:
the existing description needs some grammatical help. maybe "The extension lets you find blogs that haven't been updated recently." for the first sentence? the second sentence looks fine.